rmgpy/tools/fluxdiagram.py

```diff
@@ -166,6 +166,24 @@ def generateFluxDiagram(reactionModel, times, concentrations, reactionRates, out
                 break
             if len(edges) >= maximumEdgeCount:
                 break
+
+        if superimpose and centralSpeciesList is not None:
+            nodesCopy = nodes[:]
+            for centralSpeciesIndex in centralSpeciesIndices:
+                if centralSpeciesIndex not in nodes:  # Only add central species if it doesn't already exist
+                    nodes.append(centralSpeciesIndex)
+                    # Recursively add nodes until they connect with main graph
+                    addAdjacentNodes(centralSpeciesIndex,
+                                     nodes,
+                                     edges,
+                                     speciesList,
+                                     reactionList,
+                                     maxReactionRates,
+                                     maxSpeciesRates,
+                                     reactionCount=centralReactionCount,
+                                     rad=-1,
```

Comment:
You should both comment and note in the commit that this rad=-1 is a signaling value and doesn't represent a real radius value, otherwise this kind of coding can be confusing.